<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="map_name" default="itri" doc="options: nctu, itri, nuscene" />
    <arg name="vg_size" default="0.25" />
    <arg name="use_rviz" default="false" />


    <!-- <param name="use_sim_time" value="true"/> -->
    <node name="solution1_node" pkg="midterm_localization" type="solution1_node" output="screen" >
        <param name="map_name" value="$(arg map_name)" />
        <param name="vg_size" value="$(arg vg_size)" />
    </node>

    <!-- RVIZ -->
    <group if="$(eval arg('use_rviz') == true)">
        <node pkg="rviz" type="rviz" name="rviz_node" 
            args="-d $(find midterm_localization)/rviz/default_setting.rviz"/>
    </group>

</launch>